guide/Radiance_support.rst

```diff
@@ -1,34 +1,200 @@
+DART ``mkmf.template.rttov.gfortran`` file to your own build system.  
+Refer to the other DART ``mkmf.template`` examples that best matches your compile
+setup if you are not using gfortran.
+
+**3. Modify input.nml**
+
+Go into the work directory (e.g. ``${DART_install}/models/wrf/work``) for your 
+atmospheric model of choice. These instructions use the WRF model as an example.
+Add the appropriate  observation types for your application listed in
+``obs_def_rttov_mod.f90`` to the ``input.nml`` namelist 
+(either to ``assimilate_these_obs_types`` or ``evaluate_these_obs_types``).
+
+Include the RTTOV observation operators within the build process by editing
+the ``input_files`` and ``quantity_files`` namelist variables in the
+ ``&preprocess`` section of the ``input.nml`` file:
+
+
+.. code-block:: bash
+
+   &preprocess_nml
+      quantity_files          =  '../../../assimilation_code/modules/observations/atmosphere_quantities_mod.f90',
+                                 '../../../assimilation_code/modules/observations/ocean_quantities_mod.f90',
+                                 '../../../assimilation_code/modules/observations/chemistry_quantities_mod.f90',
+                                 '../../../assimilation_code/modules/observations/land_quantities_mod.f90'
+      input_files              = '../../../observations/forward_operators/obs_def_reanalysis_bufr_mod.f90',
+                                 '../../../observations/forward_operators/obs_def_radar_mod.f90',
+                                 '../../../observations/forward_operators/obs_def_metar_mod.f90',
+                                 '../../../observations/forward_operators/obs_def_dew_point_mod.f90',
+                                 '../../../observations/forward_operators/obs_def_rel_humidity_mod.f90',
+                                 '../../../observations/forward_operators/obs_def_gts_mod.f90',
+                                 '../../../observations/forward_operators/obs_def_rttov_mod.f90',
+
+
+**4. For your model of choice, run ./quickbuild.sh.**
+
+.. _workflow:
+
+High-level workflow
+-------------------
+
+Prior to performing a perfect model experiment (PMO) or real data assimilation using
+the DART executables ``./perfect_model_obs`` or ``./filter``, you need to link 
+the RTTOV coefficient files to the expected coefficient filename in 
+the work directory of your model.
+An example of coefficent files is provided below. These should be customized
+for your application:
+
+.. code-block:: bash
 
-DART now includes the ability to use the RTTOV forward operators for
-satellite radiance assimilation. This is a new capability for DART,
-please submit issues with the `DART
-Issues <https://github.com/NCAR/DART/issues>`__ facility.
+   export DART=${DART_install}
+   export RTTOV=${RTTOV_install}
+   cd $DART/models/wrf/work/
+   ln -sf $DART/observations/forward_operators/rttov_sensor_db.csv .
+   ln -sf $RTTOV/rtcoef_rttov13/cldaer_visir/sccldcoef_msg_4_seviri.dat  .
```

Comment:
rtcoef_rttov13, however the build instructions are for rttov12 (obs_def_rttov_mod.f90)